kolibri/plugins/coach/assets/src/views/lessons/LessonResourceSelectionPage/ContentCardList.vue

```diff
@@ -1,25 +1,32 @@
 <template>
 
-  <div>
-    <ul class="content-list">
-      <KCheckbox
-        v-if="showSelectAll"
-        :label="$tr('selectAllCheckboxLabel')"
-        :checked="selectAllChecked"
-        :indeterminate="selectAllIndeterminate"
-        :disabled="isSelectAllDisabled"
-        @change="$emit('changeselectall', $event)"
-      />
-      <KRadioButtonGroup>
-        <li
-          v-for="content in contentList"
-          :key="content.id"
-          class="content-list-item"
-          :aria-selected="contentIsChecked(content)"
-        >
+  <div class="content-list">
+    <KCheckbox
+      v-if="showSelectAll"
+      :label="$tr('selectAllCheckboxLabel')"
+      :checked="selectAllChecked"
+      :indeterminate="selectAllIndeterminate"
+      :disabled="isSelectAllDisabled"
+      @change="$emit('changeselectall', $event)"
+    />
+    <KCardGrid layout="1-1-1">
+      <component
+        :is="content.is_leaf ? 'AccessibleResourceCard' : 'AccessibleFolderCard'"
+        v-for="content in contentList"
+        :key="content.id"
+        :to="contentCardLink(content)"
+        :contentNode="content"
+        :thumbnailSrc="content.thumbnail"
+        :headingLevel="cardsHeadingLevel"
+        :isBookmarked="isBookmarked(content.id)"
+        @toggleBookmark="toggleBookmark"
+      >
+        <template #belowTitle>
+          <p v-if="contentCardMessage(content)">{{ contentCardMessage(content) }}</p>
```

Comment:
I found a little regression here. In the SelectFromBookmarks component, the cards now dont show the Bookmarked label:

Before:
![Screenshot from 2025-02-13 13-50-49](https://github.com/user-attachments/assets/d56d1636-9583-4cdc-b1f6-958d27cc6647)

After:
![image](https://github.com/user-attachments/assets/79ec0443-120d-4aac-81e2-177dc36ab308)

Reply:
Great catch! Fixed